src/main.cpp

```diff
@@ -1543,9 +1546,18 @@ extern meshtastic_DeviceMetadata getDeviceMetadata()
 #if NO_EXT_GPIO && NO_GPS || MESHTASTIC_EXCLUDE_SERIAL
     deviceMetadata.excluded_modules |= meshtastic_ExcludedModules_SERIAL_CONFIG;
 #endif
-#ifndef ARCH_ESP32
+#if MESHTASTIC_EXCLUDE_PAXCOUNTER
```

Comment:
previous the logic here excluded Paxcounter from all arch except ESP32. Now it's included in more devices, this will have an impact - particularly on the ones with memory usage running close to the limit.